.github/workflows/schedule.yml

```diff
@@ -66,3 +66,58 @@ jobs:
 
             - name: Run the unit tests
               run: vendor/bin/phpunit.bat --colors=always
+
+    cypress:
+        name: Cypress
+        runs-on: ubuntu-latest
+        steps:
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: 8.1
+                  extensions: dom, fileinfo, filter, gd, hash, intl, json, mbstring, mysqli, pcre, pdo_mysql, zlib
+                  coverage: none
+
+            - name: Initialize the database
+              run: |
+                  sudo /etc/init.d/mysql start
+                  mysql -uroot -proot -e "CREATE database contao_test"
+
+            - name: Install Node.js
+              uses: actions/setup-node@v2
+              with:
+                  node-version: ${{ env.NODE_VERSION }}
+
+            - name: Install Symfony CLI
+              run: |
+                  wget https://get.symfony.com/cli/installer -O - | bash
+                  mv /home/runner/.symfony5/bin/symfony /usr/local/bin/symfony
+
+            - name: Checkout
+              uses: actions/checkout@v3
+
+            - name: Install the dependencies
+              run: |
+                  composer install --no-interaction --no-progress
+                  composer bin cypress install --no-interaction --no-progress
+
+            - name: Run Cypress tests
+              id: cypress
+              run: |
+                  symfony server:ca:install
+                  symfony server:start --port=8765 --dir=tools/cypress/webspace --daemon
+                  cd tools/cypress
+                  npx cypress run --project ../../ --config-file tools/cypress/cypress.config.js
+                  cd ../../
```

Comment:
```suggestion
```

I think this is not necessary, is it?

Reply:
Otherwise the `--dir=` parameter in the next line would have to be adjusted.
I thought it looks nicer this way. But you can change it as you like.

Reply:
What do you think about this?

```
                  symfony server:ca:install
                  cd tools/cypress
                  symfony server:start --port=8765 --dir=webspace --daemon
                  npx cypress run --project ../../ --config-file cypress.config.js
                  symfony server:stop --dir=webspace
```

Reply:
:+1: fine with me